datasetinsights/io/kfp_output.py

```diff
@@ -68,23 +76,31 @@ def write_metric(self):
             self.data["metrics"].append(
                 {"name": key, "numberValue": val, "format": "RAW"}
             )
-        if not os.path.exists(self.filepath):
-            os.makedirs(self.filepath)
-        with open(os.path.join(self.filepath, self.filename), "w") as f:
+        if not os.path.exists(self.kfp_log_dir):
```

Comment:
This check is scatted in two places `add_metrics` and `create_tb_visualization_json`. Can this be done when the class is initialized?

Reply:
Moved this to initialization.